WebDriverAgentLib/Commands/FBElementCommands.m

```diff
@@ -652,6 +653,33 @@ + (NSArray *)routes
   }
   return FBResponseWithOK();
 }
+  + (id<FBResponsePayload>)handleWheelSelectValue:(FBRouteRequest *)request
```

Comment:
The code formatting style must be the same for all methods

Reply:
ohh